CONTRIBUTING.md

```diff
@@ -0,0 +1,502 @@
+Fortunately, markdown cells within Jupyter notebooks support LaTeX math formatting,
+and GitHub is capable of rendering these equations when viewing the notebook.
+
+#### Implementation
+
+This section should describe the details of the implementation, with particular
+focus on *why* a certain algorithm or coding decision is made, not just what
+the decision is.  For example, the naive O(N^8) integral transformation within
+MP2 is discussed in the conventional MP2 tutorial
+[here](https://github.com/psi4/psi4numpy/blob/master/Tutorials/05_Moller-Plesset/5a_conventional-mp2.ipynb)
+before the factored O(N^5) transformation is introduced.  This section benefits
+from alternating markdown and code cells, so that the reader can execute the
+cells alongside reading about the implementation.  Again, specific equations
+within publications should be referenced with [LNFA:yy:pp] keys, which should
+match those references within the `Tutorial References` section (see
+[below](#citation-styleguide)).
+
+**Note:** Every tutorial should be accompanied by a corresponding reference
+implementation.  This is because the interactive tutorials are like a
+classroom, while the reference implementation is like a textbook.  While a more
+adventurous student may skip class to rely on reading the book, every class
+needs an accompanying textbook for supporting material.  The accompanying
+reference implementation can be as simple as reproducing the code cells in a
+single script, or potentially more complex if supplemented by helper
+functions/classes/modules.
+
+**Note:** Before committing changes on a Jupyter notebook, please restart the
+Python kernel so that minimal notebook metadata is stored.  This will help to
+prevent spurious merge conflicts generated by different notebook extensions and
+execution counts.
+
+### Value Comparison
+
+To ensure the continued health of contributed reference implementations and
+interactive tutorials as Psi4 and NumPy development continues, provide one or
+more `psi4.compare_values()` statements (documented
+[here](http://psicode.org/psi4manual/1.1/api/psi4.driver.compare_values.html?highlight=compare_values#psi4.driver.compare_values))
+toward the end of your scripts/tutorials. These can contain a hard-coded
+reference value or (if available) an equivalent pure-Psi4 computation:
+
+```python
+psi4.compare_values(expected, computed, digits, label)
+```
+
+where `expected` is the reference value, and `computed` is the final output of the
+script/tutorial.  
+
+## Styleguides
+
+### Python Styleguide
+
+All Python code should should conform to the
+[PEP8](https://www.python.org/dev/peps/pep-0008/) Python style guide
+conventions.  This code should be compliant with Python versions 2.7/3.5/3.6,
+meaning that `print` must be called as a function, i.e., `print('Compliant with
+Python 3')` and not `print 'This will break Python 3!'`.
+
+### Git Commit Messages
```

Comment:
Agree, I already plan to ignore the labels, I'm afraid. Maybe comment out in fond hope that this becomes so large a project as to need triage labeling like [cmake](https://gitlab.kitware.com/cmake/cmake/issues/)